udata/settings.py

```diff
@@ -164,13 +164,16 @@ class Defaults(object):
     # The order is important to compute parents/children, smaller first.
     HANDLED_LEVELS = tuple()
 
-    # CROQUEMORT = {
-    #     'url': 'http://localhost:8000',
-    #     'delay': 1,
-    #     'retry': 10,
-    # }
-    #
-    CROQUEMORT_IGNORE = []
+    LINKCHECKING_ENABLED = True
+    LINKCHECKING_IGNORE_DOMAINS = []
+    LINKCHECKING_CACHE_DURATION = 60 * 5  # in seconds
+    LINKCHECKING_DEFAULT_LINKCHECKER = 'no_check'
```

Comment:
I don't get why we don't put the path here too? `udata.linkchecker.backends:NoCheckLinchecker`

Reply:
This is the endpoint name

Reply:
Are these settings documented somewhere?